etna/core/models/basepage.py

```diff
@@ -230,19 +232,7 @@ class BasePageWithNonRequiredIntro(BasePage):
         ),
         features=settings.INLINE_RICH_TEXT_FEATURES,
         max_length=300,
-        blank=True,
-        null=True,
     )
 
     class Meta:
         abstract = True
-
-    content_panels = BasePage.content_panels + [FieldPanel("intro")]
-
-    search_fields = BasePage.search_fields + [
-        index.SearchField("intro", boost=3),
```

Comment:
Are we meant to have lost this?

Reply:
I have them on BasePageWithIntro and then BasePageWithRequiredIntro inherits BasePageWithIntro 😃